internal/cmd/run/run.go

```diff
@@ -3,15 +3,14 @@ package run
 import (
 	"errors"
 	"fmt"
+	http2 "github.com/saucelabs/saucectl/internal/http"
```

Comment:
This somewhat meh~ named import is only temporary until all the refactor is complete.
Once this, the only http import here will be our internal one.